contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeListItem/index.vue

```diff
@@ -267,6 +304,26 @@
           this.isTopic ? this.$emit('topicChevronClick') : this.$emit('infoClick');
         }
       },
+      matchIdToString(ids) {
+        return ids.map(i => this.translateMetadataString(camelCase(i))).join(', ');
```

Comment:
This resembles the logic in the `ResourcePanel` to me. If that's right, it'd suggest to use `Intl.FormatList` here too.